src/components/vm/overview/watchdog.jsx

```diff
@@ -227,9 +227,14 @@ export const WatchdogLink = ({ vm, idPrefix, onAddErrorNotification }) => {
     const Dialogs = useDialogs();
 
     const isWatchdogAttached = Object.keys(vm.watchdog).length > 0;
+    const isPermanent = (vm.watchdog.model === "itco");
```

Comment:
This is a magic value. Is there some doc link to that? That'd be great as a comment for our future selves.

Reply:
I didn't find libvirt documentation that outright says that it is not removable. This seems to be part of emulating the q35 chipset, which has a real watchdog in its southbridge. So I added a comment explaining that.

(There is probably a way to dig out the removability of devices by checking to which kind of bus they are connected... I will have a look at that as well.)

Reply:
> (There is probably a way to dig out the removability of devices by checking to which kind of bus they are connected... I will have a look at that as well.)

No, the "itco" watchdog hasn't any information about how it is connected:
```
 <watchdog model='itco' action='reset'>
      <alias name='watchdog0'/>
    </watchdog>
```

Trying to remove it gives this error: "Error attempting device hotunplug: Operation not supported: hot unplug of watchdog of model itco is not supported"

So I guess checking the model string is our best option.

Reply:
Thanks, that's fine - I was just wondering where it came from. The comment is enough.